_posts/2018-09-23-sr2019-announced.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: news
+title: SR2019 Competition Announced
+---
+
+We are pleased to announce the details of the 2019 Student Robotics competition!
+
+After a year's hiatus during which we've worked to ensure that Student Robotics
+will be more scalable for the future, we're pleased to announce the continuation
+of the annual competition.
+
+As before the competition cycle will start with a number of concurrent kickstart
+events at which the game will be announced and the kits handed out to teams.
+
+This year kickstart will take place on Saturday 10th November in London and
+Southampton. We are hoping to confirm other venues shortly.
+
+TODO: will we have confirmed all the venues?
+TODO: will we have assigned competitor teams to locations?
+TODO: will we have space for more teams?
+
+The competition will culminate on [TODO: DATE] at the University of Southampton.
+Teams will spend two days finessing and testing their robots as they compete
+first through a league stage and then a seeded knockout stage.
+
+Rising to the top of the knockout stages is not the only way to impress the
+organisers though, and prizes will be available in a number of other categories
```

Comment:
> ...though, and prizes...

Reads strangely to me, a semicolon sounds like it might fit here

> ...though; prizes...

Reply:
good point: c9d1824